src/components/myreserve/myreserve.js

```diff
@@ -1,6 +1,99 @@
+import { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getMyReservations, cancelReservation } from '../../redux/reservations/reservetions';
+import avatarBike from '../../icons/bike-icon.png';
+import './myreserve.css';
+
 function ReservePage() {
```

Comment:
- [ ] In all your components and functions (too many to list here), please consider using ES6 syntax.  The `function` keyword is almost no longer used in the software industry.

Reply:
- There are still some parts where you use ´function´ in an arrow function Please, consider checking this.
![image](https://user-images.githubusercontent.com/20563252/193062179-4a60a284-4aa5-4f0e-9925-8569b9f98f5b.png)